conf/Makefile.chibios

```diff
@@ -131,49 +168,40 @@ endif
 # Project, sources and paths
 #
 
-# Define project name here
-PROJECT = ap
+# Project name defined in board makefile
 
 # Imported source files and paths
-
-CHIBIOS = $(CHIBIOS_EXT)
-OPENCM3 = $(OPENCM3_EXT)
-OPENCM3_INC = $(OPENCM3)/include
-OPENCM3_LIB =  $(OPENCM3)/lib/
-
-include $(CHIBIOS_BOARD_DIR)/board.mk
-include $(CHIBIOS)/os/hal/platforms/STM32F4xx/platform.mk
+# Startup files.
+include $(CHIBIOS)/os/common/ports/ARMCMx/compilers/GCC/mk/startup_stm32f4xx.mk
```

Comment:
this makes it STM32F4xx only - better to have this configurable.

An example from our (older) solution:
In board makefile:
```
# Project specific files and paths (see Makefile.chibios for details)
CHIBIOS_BOARD_PLATFORM = STM32F4xx/platform.mk
CHIBIOS_BOARD_PORT = ARMCMx/STM32F4xx/port.mk
CHIBIOS_BOARD_LINKER = STM32F407xG.ld
```

in chibios makefile:
```
include $(CHIBIOS)/os/hal/platforms/$(CHIBIOS_BOARD_PLATFORM)
include $(CHIBIOS)/os/ports/GCC/$(CHIBIOS_BOARD_PORT)
```

so I guess something along that line should work (so we can test this in Lisa M, Discovery board etc.)